### ISSUE TITLE
Sound no longer works in Guest-Mode

### ISSUE THREAD

**Issue by aarontabor** [hcilab/processing-games#66]
When I play the game in guest mode, none of the sound works (i.e., button clicks, sound effects, music). However, when I play the game logged in with a user profile, sound works correctly as expected.

As far as I can see, no additional sound-related error messages appear when playing the game in guest-mode compared to when playing logged in with a user profile.

I recently had to re-install processing, so I suspect this may be a version-related issue. Below is the contents of my sound's `library.properties` file (found in `sketchbook/libraries/sound/` on unix-like platforms):

```
name=Sound
category=Sound
authors=The Processing Foundation
url=http://processing.org/reference/libraries/sound/index.html
sentence=Sound library based on MethCla for Processing.
paragraph=
version=7
prettyVersion=1.3.2
lastUpdated=0
minRevision=228
maxRevision=0
```

# TODO
The `library.properties` is located elsewhere on windows machines (it's still somewhere within your home directory). Can some one update this post with where the user-installed libraries are saved in windows?

**Comment by aarontabor** [comment]
If someone has sound working in guest mode, please check to see if you are using a different version of the `sound` library.

**Comment by aarontabor** [comment]
Erik has the same version of the sound library mentioned above. He is have difficulties even getting the game to launch.

**Comment by aarontabor** [comment]
I'm getting the following error message when trying to launch the game from Windows 10. I've tried Processing versions `3.2.3`, `3.2.2`, and 3.2.1`.

<pre>
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  Internal Error (0x20474343), pid=7016, tid=0x0000000000000b68
#
# JRE version: Java(TM) SE Runtime Environment (8.0_102-b14) (build 1.8.0_102-b14)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.102-b14 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# C  [KERNELBASE.dll+0x71f28]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Program Files\processing-3.2.1\processing-3.2.1\hs_err_pid7016.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Could not run the sketch (Target VM failed to initialize).
For more information, read revisions.txt and Help ? Troubleshooting.
</pre>

**Comment by aarontabor** [comment]
Based on this issue report, I was able to recreate the error with the following program:

<pre>
void setup() {
  SoundFile music;
  music = new SoundFile(this, "music/bonusLevel.wav");
}

void draw() {
}
</pre>

**Comment by aarontabor** [comment]
I think our best bet is to switch over to the Minim library. The docs for the library can be found here, and it looks it will mostly just be a find-and-replace style exercise.

**Comment by aarontabor** [comment]
I've pushed a tentative fix to `feature/minim_migration` where I've migrated our code-base from the Sound library to Minim. This should fix the "Fatal Error" issue discussed above.

Regarding the original "no volume in guest mode" issue, it turns out that it was an oversight on my part. The default audio levels in guest mode are set to 0. Adjusting the volume sliders fixed it.... 

I'll test the changes on `feature/minim_migration` on Windows tomorrow morning, and then close this issue.

**Comment by aarontabor** [comment]
Momo runs on Windows using the Minim audio library, but the `setVolume()` library function isn't supported. The player can still adjust the physical volume on their computer, but the volume sliders don't do anything.

Should we accept this as a solution and merge into `development`, or keep looking for something better?

**Comment by aarontabor** [comment]
Tested on Windows and merged into development. This issue has been fixed in commits: d37e879321dc361945c1781e3bf3f328d8daab67 and 22277e8a77c976c125659d89902f0b0080739fe2
